### DISCUSSION TITLE
Welcome to Export2D Discussions!

### DISCUSSION THREAD

**Discussion by tapnair** [tapnair/Export2D#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jmlg10** [comment]
Hi,
I don't know if you will read this post.
I have a question about ExportDXF, I've tried to export multiple Faces with the "Block" Option and get only one face without any block description.

If I select the same faces and uncheck the "combine un a single DXF" I get the 3 files... so the selection is fine.

Is there any issue with "Bloks" I, need this option in order to post-process this DXF file on a router.
I need also that the blocks will be named by the name of the component if posssible...
Many Thanks
